Update area control feature options (#43107)
home-assistant/home-assistant.github.io: source/dashboards/features.markdown

      - light
      - fan
      - switch

>   - entity_id: light.kitchen_counter

Reviewer: ```suggestion
      - switch
      - entity_id: light.kitchen_counter
```

>       - entity_id: light.kitchen_counter

Reviewer: ```suggestion
  - entity_id: light.kitchen_counter
```

  type: string
controls:
  required: true

>   description: List of controls to show on the card. The list can contain domain names like `light`, `fan`, and `switch`, or specific entities using `entity_id: <entity_id>`.

Reviewer: ```suggestion
  description: List of controls to show on the card. The list can contain domain names like `light`, `fan`, and `switch`, or mappings that specify a particular entity by using the `entity_id` key, as shown in the example above.
```
